### ISSUE TITLE
Sketch for basic page

### ISSUE THREAD

**Issue by marcustisater** [postcss/postcss.org#119]
We need to identify how an normal page will look like

Needs to be included 
* Sidebar navigation for sub pages / anchor tags
* Header adjustments without hero-image
* Content text 
* Footer

**Comment by marcustisater** [comment]
@stephenway are you up for it?

**Comment by marcustisater** [comment]
@okonet - Would you like to give this a try **when** you have some time leftover? It can be very simple and you don't need to look through the details.

Everyone is welcome to leave submissions, going to try promoting this one as well.

**Comment by okonet** [comment]
Yes, I could try give it a shot. Been quite busy arm tho. Let's see if I get some time for it.

**Comment by marcustisater** [comment]
Yeah, there is defiantly no rush here, whenever you find time man :+1:

**Comment by marcustisater** [comment]
How's your work schedule looking @okonet? Otherwise I might have a try on this one.

**Comment by okonet** [comment]
@marcustisater not very good ATM. If you have some time now, go ahead. If you don't mind I could do an art direction or jump in as we have something.

**Comment by marcustisater** [comment]
Sounds good. Is it cool if I send you over the sketch for some feedback once I'm done?

**Comment by okonet** [comment]
Totally!

**Comment by marcustisater** [comment]
I forgot about this, and I do not have the time or skills to do this, hehe. It's up for grabs 😄
